Application/Services/ProjectService.cs

```diff
@@ -46,11 +46,12 @@ public async Task<ProjectResponse> UpdateAsync(Guid id, ProjectRequest projectRe
 
     public async Task DeleteAsync(Guid id)
     {
-        var tasks = await taskRepository.GetAllAsync(new TaskSearchParams(){ProjectId = id});
+        var tasks = await taskRepository.GetAllAsync(new TaskSearchParams() { ProjectId = id });
```

Comment:
Why do we need to delete all tasks in that way?

Reply:
fixed